src/events/scraper/load-cache/_get-local-date-from-filename.js

```diff
@@ -19,6 +19,9 @@ module.exports = function getLocalDateFromFilename (filename, tz='America/Los_An
    */
   file = file.substr(0, 24)
 
+  if (!/\d{4}-\d{2}-\d{2}t\d{2}_\d{2}_\d{2}\.\d{3}[Zz]/.test(file))
```

Comment:
Does not test the complete filename, only up through the key name, so we're still missing keyname, hash, ext + .gz; without the keyname and ext (and .gz) things will break.

Reply:
Right, what on earth was I thinking.  Thanks!

Reply:
@ryanblock I think we agreed that this was good to be merged as-is, with just the datetime parse check.

Conflicts resolved.